client/src/controllers/ActionController.ts

```diff
@@ -19,6 +19,7 @@ import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
 export class ActionController extends Controller<
   HTMLButtonElement | HTMLInputElement
 > {
+  static targets = ['element'];
```

Comment:
```suggestion
```

There is no need to create an element target, `this.element` is the controlled element.

See controller instance properties - https://stimulus.hotwired.dev/reference/controllers#properties